lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -167,7 +167,21 @@ def write(chunk)
           'entries' => [],
         }
         arr.each do |time, record|
-          if (record.has_key?('timeNanos'))
+          if (record.has_key?('timestamp') &&
+              record['timestamp'].has_key?('seconds') &&
+              record['timestamp'].has_key?('nanos'))
```

Comment:
If a user wants to only use seconds can he make a config that will automatically set "nanos" to zero?
If not, then maybe treat unset "nanos" as 0?